### PR TITLE
[RND-378] Ed-Fi Sample Data Analysis Notebook

### PR DESCRIPTION
**Pull request by stephenfuqua**
Before merging, I need to take another pass through the docstrings to fill in missing documentation and correct documentation that has diverged from code due to refactorings.

### CODE REVIEW

File: Equity-Analysis.ipynb [hunk 1]
```diff
@@ -0,0 +1,675 @@
...
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "setup_analysis_options()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
```
Comment (andonyns) [inline Equity-Analysis.ipynb:641]: Line 94 refers to cell 3, but there's no cell with that ID. I think that's missing here

Comment (stephenfuqua) [reply Equity-Analysis.ipynb:641]: From an [end-user perspective](https://github.com/Ed-Fi-Exchange-OSS/Sample-Data-Equity-Analysis/blob/RND-378/Equity-Analysis.ipynb) there is no notion of a "cell ID". If you're looking at the notebook, there are three code cells, so I'm referring to the last one. I agree that might not be the best way to direct users, but I didn't think of anything better right now.

File: Equity-Analysis.ipynb [hunk 2]
```diff
@@ -0,0 +1,675 @@
...
+    "Lock, Patti Frazer, et al. [Statistics: Unlocking the Power of Data](https://www.lock5stat.com/).\n",
+    "Third Edition. United States, Wiley, 2021.\n",
+    "\n",
+    "SciPy [API Reference](https://docs.scipy.org/doc/scipy/reference/index.html)\n",
+    "\n",
+    "\n",
+    "## Usage\n",
+    "\n",
+    "1. Requires Python 3.9 or 3.10 and [Poetry](https://python-poetry.org/).\n",
+    "1. You must have write access to a copy of an Ed-Fi ODS database, version 3.0 or newer.\n",
```
Comment (andonyns) [inline Equity-Analysis.ipynb:77]: Please include the instructions to install and use the poetry virtual environment

File: Equity-Analysis.ipynb [hunk 3]
```diff
@@ -0,0 +1,675 @@
...
+    "\n",
+    "Lock, Patti Frazer, et al. [Statistics: Unlocking the Power of Data](https://www.lock5stat.com/).\n",
+    "Third Edition. United States, Wiley, 2021.\n",
+    "\n",
+    "SciPy [API Reference](https://docs.scipy.org/doc/scipy/reference/index.html)\n",
+    "\n",
+    "\n",
+    "## Usage\n",
+    "\n",
+    "1. Requires Python 3.9 or 3.10 and [Poetry](https://python-poetry.org/).\n",
```
Comment (andonyns) [inline Equity-Analysis.ipynb:76]: Please include the instructions to install and use the poetry virtual environment

**Review by andonyns — CHANGES_REQUESTED** [review]

File: Equity-Analysis.ipynb (hunk 3 above)
Comment (stephenfuqua) [reply Equity-Analysis.ipynb:76]: I've added instructions on this to the readme,

**Review by andonyns — APPROVED** [review]

**Pull request merged**